libbeat/scripts/generate_fields_docs.py

```diff
@@ -70,8 +70,21 @@ def fields_to_asciidoc(input, output, beat):
 
 
     docs = yaml.load(input)
+
+    # fields file is empty
+    if docs is None:
+        return
```

Comment:
I like the idea of printing a useful error message. I think that should be combined with a useable exit code. If the script is invoked from a Makefile I would want _make_ to be able to propogate the failure. If you want to ignore the return code in _make_ then you can could prepend the command with a `-` in the makefile.